client/src/pages/Home.tsx

```diff
@@ -0,0 +1,219 @@
+import {
+  Circle,
+  GoogleMap,
+  Marker,
+  useJsApiLoader,
+} from "@react-google-maps/api";
+import React, {
+  FormEvent,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from "react";
+import "react-simple-typewriter/dist/index";
+
+import Navbar from "../components/Navbar";
+import { TripContext } from "../contexts/TripContext";
+import { CircleData } from "../dataObjects/CircleData";
+import { PlaceData } from "../dataObjects/PlaceData";
+import { PlanMarkerData } from "../dataObjects/PlanMarkerData";
+import { HomeProps } from "../types/HomeTypes";
+import {
+  radius as DEFAULT_RADIUS,
+  EPlaces,
+  keys,
+} from "../constants/GoogleMaps/config";
+import nearbySearch from "../api/GoogleMaps/nearbySearch";
+import SearchResults from "../components/Home/SearchResults";
+import TripWindow from "../components/Home/TripWindow";
```

Comment:
created two components `components/Home/SearchResults.tsx` and `components/Home/TripWindow.tsx` to house respective concerns